### PR TITLE
Add pygame 1.9.3 and dependencies

### PR DESCRIPTION
**Pull request by takluyver**
This follows from [my other pull request](https://github.com/flathub/flathub/pull/424) where I tried to make a pygame baseapp, but I was told that it was more appropriate as a shared module. So, here it is. :-)

I've currently left four new dependencies of pygame as internal modules, but I'm happy to split these out as separate modules if that's better. These are audiofile, libmikmod, fluidsynth and portmidi.

### CODE REVIEW

File: pygame/pygame-1.9.3.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                {
+                    "type": "patch",
+                    "path": "fluidsynth-no-rawmidi.patch"
+                }
+            ]
+        },
+        {
+            "name": "portmidi",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
```
Comment (TingPing) [inline pygame/pygame-1.9.3.json]: Add `-DCMAKE_BUILD_TYPE=Release` do all cmake projects.

File: pygame/pygame-1.9.3.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            "sha256": "751021819bdc0cbe5cbd51904abb6ff9e9aee5b0e8955af02284d0e77d6c9ec2"
+        },
+        {
+            "type": "patch",
+            "path": "pygame-add-search-dirs.patch"
+        }
+    ],
+    "buildsystem": "simple",
+    "build-commands": [
+      "/usr/bin/pip3 install --ignore-installed --no-deps --prefix=/app ."
```
Comment (TingPing) [inline pygame/pygame-1.9.3.json]: Don't use absolute path in the off chance somebody installs custom `pip3`.

**Comment by takluyver** [comment]
Thanks, fixed up those points. :-)

**Comment by barthalion** [comment]
I think it's fine to keep dependencies internal for now. Thanks!

**Comment by takluyver** [comment]
Thanks!

**Pull request merged**